samples/algorithms/repeat-until-success/RepeatUntilSuccess.qs

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+namespace Microsoft.Quantum.Samples.RepeatUntilSuccess {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Preparation;
+    open Microsoft.Quantum.Diagnostics;
+
+    /// # Summary
+    /// Example of a Repeat-until-success algorithm implementing a circuit 
+    /// that achieves exp(i⋅ArcTan(2)⋅Z) by Paetznick & Svore. 
+    /// Gate exp(i⋅ArcTan(2)⋅Z) is also know as V gate.
+    /// # References
+    /// - [ *Adam Paetznick, Krysta M. Svore*,
+    ///     Quantum Information & Computation 14(15 & 16): 1277-1301 (2014)
+    ///   ](https://arxiv.org/abs/1311.1074)
+    /// For circuit diagram, see file RUS.png.
+    ///
+    /// # Input
+    /// ## inputBasis
+    /// Pauli basis to prepare input qubit in
```

Comment:
It looks like this is incomplete?

Reply:
I don't think so, but the grammar is a bit awkward so I can fix that.